views/spaces/index.erb

```diff
@@ -1,24 +1,20 @@
-<style>
-	h2 {
-		text-align: center;
-		margin-top: 100px;
-	}
+<head>
+	<title>Book a Space</title>
+ <link rel="stylesheet" href="/style.css"/>
+</head>
 
-	div {
-		text-align: center;
-		margin-top: 50px;
-	}
-</style>
+<body>
+	<a href='/sessions/logout'>Sign out</a>
```

Comment:
Indentation here seems inconsistent - 4 spaces instead of 2.

Reply:
Good spot, we'll change it!